resources/js/pages/profile/components/CurrentLessonOverview.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box, Button, Flex, Heading, Text } from "@chakra-ui/react";
+
+const CurrentLessonOverview = () => {
```

Comment:
Use generic name or general name
```suggestion
const LessonOverview = () => {
```

Reply:
Done